e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -54,4 +57,26 @@ describe('Customization', () => {
             expect(config.TeamSettings.SiteName).to.eq(siteName);
         });
     });
+
+    it('SC20339 - Can change Enable Custom Branding setting', () => {
+        cy.findByTestId('TeamSettings.EnableCustomBrand').should('be.visible').within(() => {
+            // * Verify that setting is visible and matches text content
+            cy.get('label:first').should('be.visible').and('have.text', 'Enable Custom Branding: ');
+
+            // * Verify that help setting is visible and matches text content
+            const content = 'Enable custom branding to show an image of your choice, uploaded below, and some help text, written below, on the login page.';
+            cy.get('.help-text').should('be.visible').and('have.text', content);
+
+            // # Set Enable Custom Branding to true
+            cy.get('[id="TeamSettings.EnableCustomBrandtrue"]').check();
+        });
+
+        // # Click Save button
+        cy.get('#saveSetting').click();
+
+        // * Verify that the value is save, directly via REST API
+        cy.apiGetConfig().then((response) => {
+            expect(response.body.TeamSettings.EnableCustomBrand).to.equal(true);
+        });
```

Comment:
Not mentioned in the ticket but could you add test when clicking "false", something like:
```javascript
        // # Set Enable Custom Branding to false
        cy.findByTestId('TeamSettings.EnableCustomBrandfalse').check();

        // # Click Save button
        cy.get('#saveSetting').click();

        // * Verify that the value is save, directly via REST API
        cy.apiGetConfig().then((response) => {
            expect(response.body.TeamSettings.EnableCustomBrand).to.equal(false);
        });
```